src/components/ExportXprvModal.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <el-dialog
+    class="info"
+    :title="$t('close_session_info_title')"
+    width="50%"
+    :visible="true"
+    :show-close="true"
+    :close-on-click-modal="false"
+    @close="close"
+  >
+    <div slot="title" class="title-container">
+      <font-awesome-icon class="icon" icon="info-circle" />
+      <p class="title">{{ $t('export_xprv_title') }}</p>
```

Comment:
You need to add the copy to locales files

Reply:
`export_xprv_title` was wrong. Te correct key is `export_xprv_modal_title`